tests/src/Functional/Integration/PasswordPolicyTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\Tests\thunder\Functional\Integration;
+
+use Drupal\Tests\thunder\Functional\ThunderTestBase;
+
+/**
+ * Tests password policy integration.
+ *
+ * @group Thunder_password_policy
+ */
+class PasswordPolicyTest extends ThunderTestBase {
+
+  protected static $modules = [
+    'password_policy_character_types',
+    'password_policy_history',
+    'password_policy_length',
+    'password_policy',
+  ];
+
+  /**
+   * Tests redirect from old URL to new one.
+   */
+  public function testPasswordPolicies() {
+    $editor = $this->drupalCreateUser();
+    $editor->addRole('editor');
+    $editor->save();
```

Comment:
You can use $this->logWithRole() instead

Reply:
nice!

Reply:
No I can't I need the $editor, which I do not get from logWithRole()

Reply:
Then just let logWithRole() return the created user